grid/app/main/sfl/tasks/cycle.py

```diff
@@ -1,6 +1,6 @@
 import traceback
```

Comment:
The grouping "standard python, external module, local" is not being acknowledge here.